src/ColorUtils.php

```diff
@@ -2,33 +2,134 @@
 
 namespace CommandString\Utils;
 
-class ColorUtils {
-    public static function RGBAtoHEX(int $red, int $blue, int $green, ?int $alpha = null) {
-        $hex = [dechex($red), dechex($blue), dechex($green)];
-
-        if (!is_null($alpha)) {
-            $hex[] = dechex($alpha);
-        }
+/**
+ * ColourUtility is a PHP class that provides a set of utility functions for working with colours.
+ * 
+ * @method static array hexToRgb(string $hex)
+ * @method static string rgbToHex(int $r, int $g, int $b, $withPrefix = true)
+ * @method static int getBrightness(string $colour)
+ * @method static bool isDark(string $colour)
+ * @method static string getContrastColour(string $colour)
+ * @method static string getComplementaryColour(string $colour)
+ */
+class ColourUtility {
+  
+  /**
+   * The function converts a hexadecimal colour code to its corresponding RGB values.
+   * 
+   * @param string hex The "hex" parameter is a string representing a hexadecimal colour code.
+   * 
+   * @return array an array containing the RGB values of a given hexadecimal colour code. The array
+   * contains three elements, representing the red, green, and blue values respectively.
+   */
```

Comment:
Personally don't believe in commenting code. Only for intellesense purposes. And since these functions have types this isn't needed.
```suggestion
```